modules/biobank/php/pool.class.inc

```diff
@@ -0,0 +1,266 @@
+    /**
+     * Sets the pool's ID
+     *
+     * @param int $id the pool's ID
+     *
+     * @return void
+     */
+    public function setId(?int $id = null) : void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * Gets the pool's ID
+     *
+     * @return int
+     */
+    public function getId() : ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Sets the label of the pool
+     *
+     * @param string $label the pool's label
+     *
+     * @return void
+     */
+    public function setLabel(string $label) : void
+    {
+        $this->label = $label;
+    }
+
+    /**
+     * Gets the label of the pool
+     *
+     * @return string
+     */
+    public function getLabel() : string
+    {
+        return $this->label;
+    }
+
+    /**
+     * Sets the Specimen IDs of the pool
+     *
+     * @param array $specimenIds Specimen IDs of the pool
+     *
+     * @return void
+     */
+    public function setSpecimenIds(array $specimenIds) : void
+    {
+        $this->specimenIds = $specimenIds;
+    }
+
+    /**
+     * Gets the Specimen IDs of the pool
+     *
+     * @return array
+     */
+    public function getSpecimenIds() : array
+    {
+        return $this->specimenIds;
+    }
+
+    /**
+     * Sets the Candidate ID of the pool
+     *
+     * @param int $candidateId Candidate ID of the pool
+     *
+     * @return void
+     */
+    public function setCandidateId(int $candidateId) : void
+    {
+        $this->candidateId = $candidateId;
+    }
+
+    /**
+     * Gets the Candidate ID of the pool
+     *
+     * @return int
+     */
+    public function getCandidateId() : int
+    {
+        return $this->candidateId;
+    }
+
+    /**
+     * Sets the Session ID of the pool
+     *
+     * @param int $sessionId Session ID of the pool
+     *
+     * @return void
+     */
+    public function setSessionId(int $sessionId) : void
+    {
+        $this->sessionId = $sessionId;
+    }
+
+    /**
+     * Gets the Session ID of the pool
+     *
+     * @return int
+     */
+    public function getSessionId() : int
+    {
+        return $this->sessionId;
+    }
+
+    /**
+     * Sets the Center ID of the pool
+     *
+     * @param array $centerId Center ID of the pool
+     *
+     * @return void
+     */
+    public function setCenterId(int $centerId) : void
+    {
+        $this->centerId = $centerId;
+    }
+
+    /**
+     * Gets the Center ID of the pool
+     *
+     * @return int
+     */
+    public function getCenterID() : int
+    {
+        return $this->centerId;
+    }
+
+    /**
+     * Sets the date of pool
+     *
+     * @param string $date the date of the pool
+     *
+     * @return void
+     */
+    public function setDate(string $date) : void
+    {
+        $this->date = $date;
+    }
+
+    /**
+     * Gets the date of pool
+     *
+     * @return string
+     */
+    public function getDate() : string
+    {
+        return $this->date;
+    }
+
+    /**
+     * Sets the time of pool
```

Comment:
What does this mean exactly? When it was created? Should this maybe use DateTime instead?

Reply:
This is the time at the which the pool was created as inputted by the user. Perhaps a DateTime could be used instead. I'm not too familiar with DateTime so I would have to look into it. There was a point we were considering combining date and time into a single attribute, but the decision was made to keep them separate. Maybe that can be revisited at this point if you think it would be better to combine them. 

@ridz1208 thoughts?

Reply:
@HenriRabalais do you remember why we kept them separate ?

Reply:
I think the datetime html input was hard to deal with. It might be worth revisiting. But maybe not. I think it sort of makes sense to keep date and time separate.

Reply:
I clarified the comment and will implement datetime throughout a the code in a future commit.